src/packages/v4/hooks/useV4WalletHasPermission.ts

```diff
@@ -0,0 +1,40 @@
+import * as constants from '@ethersproject/constants'
+import { ProjectMetadataContext } from 'contexts/ProjectMetadataContext'
+import { useWallet } from 'hooks/Wallet'
+import { useReadJbPermissionsHasPermissions, useReadJbProjectsOwnerOf } from 'juice-sdk-react'
+import { useContext } from 'react'
+import { isEqualAddress } from 'utils/address'
+import { V4OperatorPermission } from '../models/v4Permissions'
+
+export function useV4WalletHasPermission(
+  permission: V4OperatorPermission | V4OperatorPermission[],
+): boolean {
+  const { userAddress } = useWallet()
+
+  const { projectId } = useContext(ProjectMetadataContext)
+  const _projectId = BigInt(projectId ?? 0)
```

Comment:
Could we just use projectId from an SDK context? Then we don't have to convert back to bigint

Reply:
Oh yep, which would this be? 😅